src/giterm/giterm.py

```diff
@@ -11,13 +11,13 @@
 import giterm.rungit as run
 import giterm.cursutils as cu
 
+from giterm.colors import colors
 from giterm.gui import GitermPanelManager
 from giterm._version import __version_text__
 
-
 def keyloop(stdscr):
     panels = GitermPanelManager(stdscr)
-    active = panels['history'].activate()
+    panels.active = 'history'
```

Comment:
Nice idea, though I'd prefer avoid any ambiguity with the `panels` package. What about _ui_panels_? Feel free to find a new one.